app/preference.go

```diff
@@ -12,20 +13,18 @@ import (
 func (a *App) GetPreferencesForUser(userId string) (model.Preferences, *model.AppError) {
 	preferences, err := a.Srv().Store.Preference().GetAll(userId)
 	if err != nil {
-		err.StatusCode = http.StatusBadRequest
-		return nil, err
+		return nil, model.NewAppError("GetAlGetPreferencesForUser", "app.preference.get_all.app_error", nil, err.Error(), http.StatusBadRequest)
```

Comment:
Shouldn't this be "GetPreferencesForUser"?

Reply:
:ok: